carbon/db.py

```diff
@@ -101,8 +105,26 @@ def __call__(self) -> Engine:
         )
         raise AttributeError(nonconfigured_engine_error_message)
 
-    def configure(self, conn: str) -> None:
-        self._engine = self._engine or create_engine(conn)
+    def configure(self, conn: str, **kwargs: Any) -> None:  # noqa: ANN401
+        self._engine = self._engine or create_engine(conn, **kwargs)
+
+    def run_connection_test(self) -> None:
```

Comment:
Docstrings would be beneficial for clarifying what's happening here

Reply:
Docstring has been added. ✍🏼